.docs/README.md

```diff
@@ -134,27 +123,24 @@ $this->template->channels = $service->getChannels();
 {/foreach}
 ```
 
-Volání `getChannels()` je dobré obalit zachytáváním výjimky `GopayFatalException`,
-protože napoprvé se v ní provádí dotaz na Gopay server kvůli získání výchozího
-seznamu.
+It's a good idea to wrap the `getChannels ()` call by catching the` GopayFatalException` exception because for the first time it queries the Gopay server to get the default list.
```

Comment:
It's a good idea to wrap the `getChannels()` call by catching the` GopayFatalException` exception because for the first time it queries the Gopay server to get the default list.